build.gradle

```diff
@@ -19,7 +19,7 @@ subprojects {
     ext {
         autoValueVersion = '1.7.4'
         googleCloudVersion = '1.0.2'
-        openTelemetryVersion = '0.7.0'
+        openTelemetryVersion = '0.8.0'
         junitVersion = '4.13'
         opentelemetryOperationsVersion = "0.7.1"  // CURRENT_RELEASE_VERSION
```

Comment:
@zoercai FYI we will need to bump this version to v0.8.0 and update example to use it, since we want to have our version match the SDK one.

For now the example will keep using the latest release, so no need to change anything in this PR.